tests/playwright/sanity/modules/promotions/promotions.test.ts

```diff
@@ -103,41 +103,56 @@ test.describe( 'Promotion tests @promotions', () => {
 		await expect( promoContainer ).toHaveScreenshot( `navigator-footer.png` );
 	} );
 
+	test( 'Promotions - Free to Pro - Navigator - Dark Mode', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdminPage = new WpAdminPage( page, testInfo ),
```

Comment:
Does this test also need anything about editorv2 like the others have, or it doesn't need?

Reply:
no need

Reply:
@wittytzipporah 
If a promotion is not related to the top bar than there is not need to test V2